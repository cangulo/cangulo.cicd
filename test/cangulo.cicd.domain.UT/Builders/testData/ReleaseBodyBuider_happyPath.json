{
    "input": {
        "changes": [
            "change 1",
            "change 2",
            "change 3"
        ],
        "version": "1.1.0"
    },
    "expected": ""
}