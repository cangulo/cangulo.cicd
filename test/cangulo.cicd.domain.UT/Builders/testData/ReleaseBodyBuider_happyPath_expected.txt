<!-- START:1.1.0 -->
# 1.1.0

* change 1
* change 2
* change 3

<!-- END:1.1.0 -->
